crates/host/src/wasmbus/mod.rs

```diff
@@ -1612,9 +1725,9 @@ impl Host {
     #[instrument(skip(self))]
     async fn fetch_actor(&self, actor_ref: &str) -> anyhow::Result<wasmcloud_runtime::Actor> {
         let actor = fetch_actor(
-            &actor_ref,
-            &self.host_config.oci_opts,
+            actor_ref,
             self.host_config.allow_file_load,
+            &*self.registry_settings.read().await,
```

Comment:
Could we extract this onto a separate line for clarity? (I.e. `let registry_settings = self.registry_settings.read().await`)